package/src/skia/__tests__/setup.ts

```diff
@@ -10,10 +10,9 @@ beforeAll(async () => {
   Skia = JsiSkApi(CanvasKit);
 });
 
-const width = 256;
-const height = 256;
-
 export const setupSkia = () => {
+  const width = 256;
```

Comment:
Why are we moving them inside here? Doesn't really matter but I was just wondering :)

Reply:
no reasons